docs/data/migration/migration-data-grid-v6/migration-data-grid-v6.md

```diff
@@ -31,6 +35,22 @@ To have the option of using the latest API from `@mui/material`, the package pee
 It is a change in minor version only, so it should not cause any breaking changes.
 Please update your `@mui/material` package to this or a newer version.
 
+## Update the license package
+
+If you're using the commerical versions of the Data Grid ([Pro](/x/introduction/licensing/#pro-plan) and [Premium](/x/introduction/licensing/#premium-plan) plans), you need to update the license package to the latest version.
```

Comment:
Are we sure about this? 🤔 
The license package is a direct dependency of the commercial packages, the latest version will be downloaded if they are using the latest package version. 🤔 
If we are referring to cases, where people are adding this dependency due to their package manager behavior, then maybe can make this section optional and move it below the import update, which is a clear BC for everyone? 🤔